exercises/armstrong-numbers/canonical-data.json

```diff
@@ -0,0 +1,86 @@
+{
+  "exercise": "armstrong-number",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "0 is an Armstrong number",
+      "property": "armstrong",
+      "input": {
+        "number": 0
+      },
+      "expected": true
+    },
+    {
+      "description": "1 is an Armstrong number",
+      "property": "armstrong",
+      "input": {
+        "number": 1
+      },
+      "expected": true
+    },
+    {
+      "description": "9 is an Armstrong number",
+      "property": "armstrong",
+      "input": {
+        "number": 9
+      },
+      "expected": true
+    },
+    {
+      "description": "10 is not an Armstrong number",
```

Comment:
These test descriptions are still not telling me anything about the tests.
Why is 10 a useful number to test?

Reply:
I think it's a useful test since it is the only test for 2 digit numbers and since there seems to be no armstrong 2-digit armstrong numbers it is a representative of its class.

Reply:
Then the description should include this information.
```text
"description": "There are no 2 digit Armstrong numbers",
```

Reply:
I'd expect a test with that description to iterate over all 2 digit numbers and ensure none of them are Armstrong.

Perhaps `"description": "Two digit number which is not an Armstrong number"`?

Reply:
Is there a way i can technically specify it in this json format??